test/modules/DirectMethodSender/DirectMethodCloudSender.cs

```diff
@@ -45,7 +45,12 @@ sealed class DirectMethodCloudSender : DirectMethodSenderBase
             ulong directMethodCount,
             CancellationToken none)
         {
-            CloudToDeviceMethod cloudToDeviceMethod = new CloudToDeviceMethod(methodName).SetPayloadJson($"{{ \"Message\": \"Hello\", \"DirectMethodCount\": \"{directMethodCount.ToString()}\" }}");
+            TimeSpan timeouts = Settings.Current.DirectMethodDelay;
+            CloudToDeviceMethod cloudToDeviceMethod = new CloudToDeviceMethod(
+                methodName,
+                Settings.Current.DirectMethodDelay,
```

Comment:
Why are we passing the DirectMethodDelay parameter to ConnectionTimeout and MethodTimeout?

Reply:
Its for this test bug: 
https://msazure.visualstudio.com/One/_sprints/taskboard/IoT-Platform-Edge/One/IoT/2004/2004.2?workitem=6418885

I think that DirectMethodDelay period should be enough for the re-auth to happen and pass the test.